R/add_keep_population_flag.R

```diff
@@ -0,0 +1,163 @@
+#' Add keep_popluation flag
+#'
+#' @description Add keep_population flag to individual files
+#' @param data A data frame
+#'
+#' @return A data frame with keep_population flags
+#' @family individual_file
+
+add_keep_population_flag <- function(individual_file, year) {
+  calendar_year <- paste0("20", substr(year, 1, 2)) %>% as.integer()
+
+  if (!check_year_valid(year, "NSU")) {
+    individual_file <- individual_file %>%
+      dplyr::mutate(keep_population = 1L)
+  } else {
+    ## Obtain the population estimates for Locality AgeGroup and Gender.
+    pop_estimates <-
+      readRDS(get_datazone_pop_path("DataZone2011_pop_est_2011_2021.rds")) %>%
+      dplyr::as_tibble() %>%
```

Comment:
```suggestion
      readr::read_rds(get_datazone_pop_path("DataZone2011_pop_est_2011_2021.rds")) 
```
Not sure why we need `dplyr::as_tibble()` here? i can read in the file okay when running this line of code. I am suggesting to use `readr::read_rds` here instead as i think we do that everywhere else for consistency